docs/superpowers/plans/2026-05-10-split-seaweedfs-system.md

```
@@ -0,0 +1,687 @@
+
+- [ ] **Step 2: Verify content unchanged**
+
+Run:
+
+```bash
+head -5 /home/daniil/aenix/cozystack-split-seaweedfs/packages/system/seaweedfs-db/templates/database.yaml
+```
+
+Expected:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)